modules/node_modules/@ncigdc/containers/CaseTr.js

```diff
@@ -35,7 +36,17 @@ export type TProps = {|
   theme: Object,
 |};
 
-export const CaseTrComponent = ({ node, index, theme }: TProps) => {
+function massageRelayFiles(files: Array): Array {
+  return _.get(files, 'hits.edges', [])
```

Comment:
>files is typed Array in the signature, should it be an Object

Ack, yup you're right. Actually tripped myself up a bit on that at first and forgot to change the signature

>  (files.hits || { edges : [] }).edges.map
Looks like it'd have to be `((files || {}).hits || { edges: [] }).edges` since `files` is sometimes passed in as `undefined`. That looks a bit harder to parse than `_.get(files, 'hits.edges', [])` imo

Don't mind changing that to lodashless if you feel strongly about it, but would much prefer to take a look at our jest setup to see if we can fix the lodash2 issue there :p  What do you think?